contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+
+    getPathsForAllAnswers() {
+        const allAnswerPaths = []
+        const allAnswers = []
+        const answersUsed = []
+
+        // Log ids of all answers in section
+        this.questions.forEach((question) => {
```

Comment:
IMO this would be cleaner if you removed lines 165-169, and replaced line 161 with

```js
const allAnswers = questions.flatMap(question => question.answers.map(answer => answer.id));
```

Reply:
In addition to cleaner, it _should_ be much more performant but YMMV